### ISSUE TITLE
ArFoundation v1.5 samples : Error when using Assembly Definition files, Arsubsystems reference is missing

### ISSUE THREAD

**Issue by St4rwolf13** [Unity-Technologies/arfoundation-samples#144]
Hello everyone, i just tryed the Arfoundation samples version 1.5, when i create assembly definition file for my scripts, Arsubsystems reference doesn't appear in the list, but the package is in the files, i repaired it by adding  this line :
"com.unity.xr.arsubsystems" : "2.1.0-preview.1",

in the Packages/manifest.json file, in the dépendencies.
after this, the Arsubsystems line appear for the assembly reference.

truly sorry for my bad english.
regards

**Comment by tiancaipipi110** [comment]
@St4rwolf13 Im getting lots of missing assembly references when update to ARFoundation 1.5.0. Which Unity version are you using and what does your `manifest.json` looking like?

**Comment by St4rwolf13** [comment]
@tiancaipipi110  i tested ARfoundation 1.5.0 for my work, i used Unity 2019.1.3f1 and for the `manifest.json` it was the sample manifest (https://github.com/Unity-Technologies/arfoundation-samples/blob/1.5-preview/Packages/manifest.json), i just added this line `"com.unity.xr.arsubsystems" : "2.1.0-preview.1"` after the ARfoundation line.

**Comment by daltonbr** [comment]
Thanks for the workaround @St4rwolf13! I had the same problem when updating to ARFoundation 1.5.0, I am using Unity 2018.4.4f1.
Don't be sorry for your English, your message was helpful and very clear!👍

@tiancaipipi110 I've also had lots of missing assembly references (both in Rider and Visual Studio on Mac), still the code was working correctly even if the IDE was not getting that reference (e.g. code autocomplete didn't work)

In my current project, I am using **Assembly Definition files**, and I guess that was the reason for this problem, so after adding that extra line on the ```manifest.json``` I was able to drag the missing reference (```com.unity.xr.arsubsystems```) in Assembly Definition, and it took a while but the IDE finally gets the reference right. (I tried to rebuild the Library folder, Rebuild the solution).

<img width="351" alt="AssemblyDefinition" src="https://user-images.githubusercontent.com/10944679/61448072-7e20c500-a952-11e9-87ce-f72ba220a78b.png">
